decidim-core/app/views/decidim/devise/invitations/edit.html.erb

```diff
@@ -13,6 +13,8 @@
           <%= decidim_form_for resource, as: resource_name, url: invitation_path(resource_name, invite_redirect: params[:invite_redirect]), html: { method: :put, class: "register-form new_user" } do |f| %>
             <%= f.hidden_field :invitation_token %>
 
+            <%= f.text_field :nickname %>
```

Comment:
I think this needs a tip or something... :thinking:

Reply:
👍 on adding a tip to this